packages/manager/core/generator/app/templates/src/tracking.constant.ts.hbs

```diff
@@ -0,0 +1,4 @@
+export const LEVEL2 = '86';
+export const UNIVERSE = '{{universe}}';
+export const SUB_UNIVERSE = '{{subuniverse}}';
```

Comment:
```suggestion
export const SUBUNIVERSE = '{{subuniverse}}';
```

Remove `_` or keep it everywhere to stay consistent

Reply:
👍